sample-tests/build.gradle.kts

```diff
@@ -22,11 +23,19 @@ val tokens: Map<String, Provider<String>> = mapOf(
 
 tasks {
     val replaceTokensInSnippets by registering(Copy::class) {
-        from(snippetsDir)
         into(processedSnippetsDir)
         inputs.properties(tokens)
-        doFirst {
-            filter(org.apache.tools.ant.filters.ReplaceTokens::class, "tokens" to tokens.mapValues { it.value.get() })
+        from(snippetsDir) {
+            // First copy all non-filtered files to preserve binaries such as the gradle-wrapper.jar
+            exclude("**/*.gradle.kts")
+        }
+        from(snippetsDir) {
+            include("**/*.gradle.kts")
+            doFirst {
+                filter(
+                    org.apache.tools.ant.filters.ReplaceTokens::class,
```

Comment:
Does it make sense to import this class?

Reply:
Done in c0db5cacea75ba430ddd034dbb8fad4465b8dc92.